contracts/token/Minter.sol

```diff
@@ -26,35 +26,21 @@ contract Minter is Manager, IMinter {
     // Current number of redistributed tokens. Reset every round
     uint256 public currentRedistributedTokens;
 
-    // Sender must be RoundsManager
-    modifier onlyRoundsManager() {
-        require(msg.sender == controller.getContract(keccak256("RoundsManager")));
-        _;
-    }
-
-    // Sender must be BondingManager
-    modifier onlyBondingManager() {
-        require(msg.sender == controller.getContract(keccak256("BondingManager")));
-        _;
-    }
-
-    // Sender must be BondingManager or JobsManager
-    modifier onlyBondingManagerOrJobsManager() {
-        require(msg.sender == controller.getContract(keccak256("BondingManager")) || msg.sender == controller.getContract(keccak256("JobsManager")));
-        _;
-    }
-
     function Minter(address _controller, uint256 _initialTokenSupply, uint8 _yearlyInflation) Manager(_controller) {
         initialTokenSupply = _initialTokenSupply;
         yearlyInflation = _yearlyInflation;
     }
 
+    function setParameters(uint8 _yearlyInflation) external onlyAuthorized {
```

Comment:
Should this be `setParameters` or `setYearlyInflation`?

Reply:
I was thinking that the function used to set parameter values on each contract is the same across all contracts, but open to other thoughts